### PR TITLE
Nn 3013 return bed assignment history sequence for cell history

### PR DESCRIPTION
**Pull request by JamieDraperUK**

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/api/model/BedAssignment.java [hunk 1]
```diff
@@ -19,13 +23,33 @@
 @ToString
 @Data
 public class BedAssignment {
+    @ApiModelProperty(value = "Unique, numeric booking id.", position = 1, example = "1234134")
```
Comment (petergphillips) [inline src/main/java/uk/gov/justice/hmpps/prison/api/model/BedAssignment.java:26]: Is this unique - thought it had to be combination of `bookingId` and `badAssignmentHistorySequence`?

**Review by petergphillips — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/justice/hmpps/prison/api/model/BedAssignment.java (hunk 1 above)
Comment (JamieDraperUK) [reply src/main/java/uk/gov/justice/hmpps/prison/api/model/BedAssignment.java:26]: Good point, I'll update the docs :+1:

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**